Sources/Public/Animation/LottieView.swift

```diff
@@ -175,27 +175,56 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     return copy
   }
 
-  // Returns a copy of this view playing once from the current frame to the end frame
+  @available(*, deprecated, renamed: "playing(_:)", message: "Will be removed in a future major release.")
   public func play() -> Self {
-    play(loopMode: .playOnce)
+    playbackMode(.play(.fromProgress(nil, toProgress: 1, loopMode: .playOnce)))
   }
 
   /// Returns a copy of this view that loops its animation from the start to end whenever visible
   public func looping() -> Self {
-    play(.fromProgress(0, toProgress: 1, loopMode: .loop))
+    playbackMode(.play(.fromProgress(0, toProgress: 1, loopMode: .loop)))
   }
 
-  /// Returns a copy of this view playing from the current frame to the end frame,
-  /// with the given `LottiePlaybackMode`.
+  @available(*, deprecated, renamed: "playbackMode(_:)", message: "Will be removed in a future major release.")
   public func play(loopMode: LottieLoopMode = .playOnce) -> Self {
-    play(.toProgress(1, loopMode: loopMode))
+    playbackMode(.play(.fromProgress(nil, toProgress: 1, loopMode: loopMode)))
   }
 
-  /// Returns a copy of this view playing with the given `LottiePlaybackMode`
+  @available(*, deprecated, renamed: "playbackMode(_:)", message: "Will be removed in a future major release.")
   public func play(_ playbackMode: LottiePlaybackMode) -> Self {
     self.playbackMode(playbackMode)
   }
 
+  /// Returns a copy of this view playing with the given playback mode
+  public func playing(
+    _ playbackMode: LottiePlaybackMode
+      .PlaybackMode = .fromProgress(nil, toProgress: 1, loopMode: .playOnce))
+    -> Self
+  {
```

Comment:
```suggestion
  public func playing(
    _ playbackMode: LottiePlaybackMode.PlaybackMode = .fromProgress(nil, toProgress: 1, loopMode: .playOnce))
    -> Self
  {
```

Reply:
SwiftFormat is funny sometimes

Reply:
I think we also don't need the default: https://github.com/airbnb/lottie-ios/pull/2198/files#r1336439481